fendermint/eth/api/src/conv/from_tm.rs

```diff
@@ -360,6 +363,9 @@ where
         withdrawals_root,
         withdrawals,
         other,
+        blob_gas_used,
```

Comment:
It looks like https://eips.ethereum.org/EIPS/eip-4844 has already been implemented in ethers. Hopefully it doesn't break anything for us :crossed_fingers: